### PR TITLE
Add python-leveldb

### PR DESCRIPTION
**Pull request by jakirkham**

### CODE REVIEW

File: recipes/python-leveldb/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install
+if errorlevel 1 exit 1
+
+:: Add more build steps here, if they are necessary.
```
Comment (pelson) [inline recipes/python-leveldb/bld.bat]: I think these comments are superfluous.

**Comment by pelson** [comment]
Failing on Windows :cry: 

```
BUILD START: python-leveldb-0.193-py27_0 
Fetching package metadata: ........ 
Solving package specifications: . 
Traceback (most recent call last): 
  File "C:\Miniconda\Scripts\conda-build-all-script.py", line 9, in <module> 
    load_entry_point('conda-build-all==0.9.0', 'console_scripts', 'conda-build-all')() 
  File "C:\Miniconda\lib\site-packages\conda_build_all\cli.py", line 85, in main 
    b.main() 
  File "C:\Miniconda\lib\site-packages\conda_build_all\builder.py", line 207, in main
    built_dist_location = self.build(meta) 
  File "C:\Miniconda\lib\site-packages\conda_build_all\builder.py", line 164, in build 
    return bldpkg_path(build.build(meta.meta)) 
  File "C:\Miniconda\lib\site-packages\conda_build_all\build.py", line 32, in build
    build_module.build(meta, post=None, **kwd) 
  File "C:\Miniconda\lib\site-packages\conda_build\build.py", line 400, in build 
    [ms.spec for ms in m.ms_depends('build')]) 
  File "C:\Miniconda\lib\site-packages\conda_build\build.py", line 307, in create_env
    actions = plan.install_actions(prefix, index, specs) 
  File "C:\Miniconda\lib\site-packages\conda\plan.py", line 413, in install_actions
    pkgs = r.install(specs, [d + '.tar.bz2' for d in linked], update_deps=update_deps) 
  File "C:\Miniconda\lib\site-packages\conda\resolve.py", line 875, in install 
    pkgs = self.solve(specs, len0=len0, returnall=returnall) 
  File "C:\Miniconda\lib\site-packages\conda\resolve.py", line 909, in solve 
    dists, new_specs = self.get_dists(specs) 
  File "C:\Miniconda\lib\site-packages\conda\resolve.py", line 399, in get_dists 
    specs, removes, optional, features = self.verify_specs(specs)
  File "C:\Miniconda\lib\site-packages\conda\resolve.py", line 393, in verify_specs
    raise NoPackagesFound(bad_deps)
conda.resolve.NoPackagesFound:  Package missing in current win-32 channels: 
  - leveldb
```

Looks like the leveldb feedstock skips windows: https://github.com/conda-forge/leveldb-feedstock/blob/master/recipe/meta.yaml#L14

File: recipes/python-leveldb/bld.bat (hunk 1 above)
Comment (jakirkham) [inline recipes/python-leveldb/bld.bat]: Good point. Sorry, I forgot this one.

**Comment by jakirkham** [comment]
Yeah, that's because there is no build of `leveldb` for Windows. We can revisit Windows when that changes.

**Comment by jakirkham** [comment]
Now, we are passing. :) More thoughts on this one?

File: recipes/python-leveldb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - leveldb
+    - python
+
+test:
+  imports:
+    - leveldb
+
+about:
+  home: http://code.google.com/p/py-leveldb/
+  license: BSD License
```
Comment (jakirkham) [inline recipes/python-leveldb/meta.yaml]: This is 3-Clause also. Will update.

Comment (jakirkham) [inline recipes/python-leveldb/meta.yaml]: Fixed.

File: recipes/python-leveldb/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  run:
+    - leveldb
+    - python
+
+test:
+  imports:
+    - leveldb
+
+about:
+  home: http://code.google.com/p/py-leveldb/
```
Comment (jakirkham) [inline recipes/python-leveldb/meta.yaml]: Unfortunately, they still seem to be using the Google Code site. So, that's what we have. At some point, they will move and we can update.

**Comment by jakirkham** [comment]
Any more comments or are we ok with a merge here?

**Comment by ocefpaf** [comment]
Remember to delete the `appveyor.yml` in the feedstock :wink:

**Comment by jakirkham** [comment]
Thanks @ocefpaf. I think @pelson added some code to automate `appveyor.yml` deletion.

**Comment by ocefpaf** [comment]
> Thanks @ocefpaf. I think @pelson added some code to automate appveyor.yml deletion.

Yeah, but I am not sure is release yet. Let's see...

**Comment by jakirkham** [comment]
A test of the system. ;)

**Comment by jakirkham** [comment]
You're right. Not yet released.

**Pull request merged**